cms/envs/common.py

```diff
@@ -600,7 +600,13 @@
 
 # Don't use compression by default
 PIPELINE_CSS_COMPRESSOR = None
```

Comment:
Nit: Perhaps update the comment on line 601, or add a separate one to explain why we are now using the JS compression